dataGenerator/src/main/java/com/filip/datagenerator/Service/Impl/DataGeneratorImpl.java

```diff
@@ -0,0 +1,60 @@
+package com.filip.datagenerator.Service.Impl;
+
+import com.filip.datagenerator.Enum.Status;
+import com.filip.datagenerator.KafkaSenders;
+import com.filip.datagenerator.Model.Account;
+import com.filip.datagenerator.Model.Transaction;
+import com.filip.datagenerator.Service.DataGenerator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+
+@Service
+public class DataGeneratorImpl implements DataGenerator {
+
+    @Value("${OFFLINE_TRANSACTION_UPPER_LIMIT}")
+    Integer limit;
+
+    @Autowired
+    KafkaSenders sender;
+
+    @Override
+    public void insertAccounts() throws ExecutionException {
+        int n = 0;
+        while (n < 21) { // TODO: 10001
+            List<Transaction> tlist = new ArrayList<>();
+            Random random = new Random(505000);
```

Comment:
Why 505000?

Reply:
505000 + 15000 = 520000 which should be the maximum amount account could have. (look at line 33 and below) Amount is not specified in the project so for the sake of simplicity the minimum amount is 15000 and max is 520000. I can change this to something else?